app/src/main/java/com/isc/hermes/SearchViewActivity.java

```diff
@@ -97,6 +98,12 @@ public void onItemClick(WayPoint wayPoint) {
         editor.putFloat("longitude", (float)wayPoint.getLongitude());
         editor.apply();
         finish();
+        /* Optional
```

Comment:
delete these comments or if it works for you later add a TODO

Reply:
Done.